test/helpers/desk_helper_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class DeskHelperTest < ActionView::TestCase
+
+  setup do
+    @occupied_desk = desks :desk1
+    @unoccupied_desk = desks :desk2
+  end
+
+  test '#desk_status gives correct status' do
+    assert_equal(desk_status(@occupied_desk), 'Occupied')
```

Comment:
More of a styling nit: In assertions, we generally don't include the brackets, so in Ruby land `assert_equal foo, bar` is more preferable